plugins/javadoc/src/main/kotlin/javadoc/renderer/JavadocContentToTemplateMapTranslator.kt

```diff
@@ -206,6 +203,9 @@ internal class JavadocContentToTemplateMapTranslator(
             )
         }
 
+        private fun renderInlineParameter(parameter: JavadocParameterNode): String =
+            "${htmlForContentNode(parameter.type, contextNode)} ${parameter.name}"
```

Comment:
I was thinking more of a `htmlForContentNode(parameter.type, contextNode) + " ${parameter.name}"` instead but maybe that's just me